activity/index.html

```diff
@@ -0,0 +1,133 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Community Activity</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="timeago.js"></script>
+    <style>
+    canvas {
+        -moz-user-select: none;
+        -webkit-user-select: none;
+        -ms-user-select: none;
+    }
+    </style>
+</head>
+
+<body>
+    <h1>Community Activity</h1>
+    <script>loadTimeElements()</script>
+    <br>
+    <br>
+    <div style="width:75%;">
+        <canvas id="canvas"></canvas>
+    </div>
+    <select id="chartType">
+        <option value="Month">Year</option>
+        <option value="Week">Month</option>
+        <option value="Day">Week</option>
+    </select>
+    <br>
+    <br>
+    <script>
+    var openedData = []
+    var closedData = []
+    var labels = [];
+    var myChart;
+
+    function updateChart() {
```

Comment:
this should also be in static JS files, so they can be linted.

pass the document elements into the functions.